lessons/codap-bar-charts/langs/en-us/pages/codap-displays-intro.adoc

```diff
@@ -1,85 +1,59 @@
+Open the @starter-file{animals}. First, create a graph of randomly generated points by selecting the `Graph` icon, and then respond to the following prompts.
 
-== Creating Bar Charts in CODAP
+== Create Displays
 
-@n Select the y-axis on your graph (where it says "Click here"). On the drop-down menu that apperas, select `fixed`. (_If you prefer, you may also drag the attribute name from the table to the y-axis._) What do you notice?
+@n Select the y-axis on your graph (where it says "Click here"). On the drop-down menu that appears, select `Fixed`. (_If you prefer, you may also drag the attribute name from the table to the y-axis._) What do you notice?
 
-
-@ifsoln{A dot plot forms, categorizing animals as fixed (true) and not fixed (false). The labels for the dot plot are on the y-axis. The dots are stacked horizontally.}
-
-@ifnotsoln{
-@fitb{}{}
-
-@fitb{}{}
+@fitb{}{@ifsoln{A horizontal dot plot forms, categorizing animals as fixed (true) and not fixed (false). The labels for the dot plot are on the y-axis.}}
 
 @fitb{}{}
-}
 
 @vspace{1ex}
 
-@n Now select the x-axis on your graph and select `fixed`. How does the graph change?
+@n Now select the x-axis on your graph and select `Fixed`. How does the graph change?
 
-@ifsoln{The labels move from the y-axis to the x-axis, and the dots are stacked vertically.}
+@fitb{}{@ifsoln{The labels move from the y-axis to the x-axis, and the dots are stacked vertically.}}
 
-@ifnotsoln{
 @fitb{}{}
 
-@fitb{}{}
-}
 @vspace{1ex}
 
 @n Select the `configuration` icon (which looks like a bar graph) to the right of the data display. Select `fuse dots into bars`
 
 @n Click the `ruler` icon, and test out each of the two options available (count and percentage). What happens?
 
-@ifnotsoln{
-@fitb{}{}
+@fitb{}{@ifsoln{When I select "count," I see the number of animals represented by each bar. When I select "percentage," the percentage of }}
 
-@fitb{}{}
-}
-
-@ifsoln{When I select "count," I see the number of animals represented by each bar. When I select "percentage," the percentage of animals represented by each bar appears.}
+@fitb{}{@ifsoln{animals represented by each bar appears.}}
 
 @vspace{1ex}
 
 
 @n Now, make a bar chart showing how many animals there are of each species by changing the variable on the x-axis to `species`. Can you figure out how to reconfigure the bar chart as a dot plot?
 
-@ifnotsoln{
-@fitb{}{}
+@fitb{}{@ifsoln{To reconfigure as a dot plot, select the `configuration` icon and deselect `fuse dots into bars`.}}
 
 @fitb{}{}
-}
 
-@ifsoln{To reconfigure as a dot plot, select the `configuration` icon and deselect `fuse dots into bars`.}
 
 @vspace{1ex}
 
-== Numeric versus Categorical Displays
+== Numeric vs. Categorical Displays
 
 @n Create a graph with `weight` on the x-axis. Then, click on `weight` so that a drop-down menu appears. From this drop-down menu, choose `Treat as Categorical`, and observe how the graph changes. What is going on?
```

Comment:
noting that I changed `weight` to `Pounds` but may not have caught all instances of this so be on the lookout for others.